lib/rst/interp_float/point2d.c

```diff
@@ -31,7 +31,9 @@
 #include <grass/vector.h>
 #include <grass/dbmi.h>
 
-#define	POINT2D_C
+#ifndef POINT2D_C
```

Comment:
This change is not Cmake related and should be separated out

Reply:
Definitely not related to CMake. @rkanavath Please can you elaborate on this change?